.changeset/serious-numbers-know.md

```diff
@@ -0,0 +1,9 @@
+---
+"@chakra-ui/cli": minor
+---
+
+Allow generating theme typings to use via module augmentation
+
+```bash
+chakra-cli tokens --template augmentation
```

Comment:
**suggestion**: I think we should include the `--out` parameter in this changelog entry as well, otherwise the augmented typings will be generated in the node_modules, which might be unexpected.

```suggestion
chakra-cli tokens --template augmentation --out ./types/chakra-ui__styled-system.d.ts
```